CONTRIBUTING.md

```diff
@@ -1,21 +1,21 @@
 <!---
-  Copyright 2022 SECO Mind Srl
+  Copyright 2022-2024 SECO Mind Srl
 
   SPDX-License-Identifier: CC0-1.0
 -->
 
-# Contributing
+# Contributing to Astarte
 
-Astarte is open to any contribution.
+Thank you for your interest in contributing to Astarte!
 
-[Pull requests](https://github.com/astarte-platform/astarte/pulls),
-[bug reports and feature requests](https://github.com/astarte-platform/astarte/issues) are welcome.
+## Development Guide
 
-However before contributing, please read carefully our [Code of Conduct](CODE_OF_CONDUCT.md) and
-the following contribution guidelines.
+For detailed information on setting up your development environment, coding standards,
+submitting patches, and more, please refer to our [Astarte Development Guide](TODO: add link).
```

Comment:
The [Development guide](https://github.com/astarte-platform/astarte/pull/963) does not contain specific guidelines about coding standards, submitting patches, Developer Certificate of Origin, commit signing.

We can discuss whether to keep these guidelines and where.

Reply:
For now I've reworded it so that only information present in the dev guide is present